File: C++/SivitasAkademik.cpp

class SivitasAkademik :  public Human{
    
    private:
        string asal_univ;
        string email;

    public:
    // Constructor
        SivitasAkademik(){
            this->asal_univ = "";
            this->email = "";
        }

        SivitasAkademik(string asal_univ, string email, string NIK, string name, string gender) : Human(NIK, name, gender){
            this->asal_univ = asal_univ;
            this->email = email;
        }

    // Setter dan Getter

        // Untuk Asal Universitas
        string get_Asal_univ(){
            return this->asal_univ;
        }

        void set_Asal_univ(string asal_univ){
            this->asal_univ = asal_univ;
        }

        // Untuk Email
        string get_Email(){
            return this->email;
        }

        void set_Email(string email){
            this->email = email;
        }

        // Destruktor
        ~SivitasAkademik(){
        }
    
};

File: PHP/SivitasAkademik.php
<?php

class SivitasAkademik extends Human{

	// Atribut private dari Class SivitasAkademik
	private $asal_univ = "";
    private $email = "";
	
	// Atribut public dari Class SivitasAkademik
	public function __construct($asal_univ = "", $email = "", $NIK = "", $name = "", $gender = ""){
		parent::__construct($NIK, $name, $gender);
        $this->asal_univ = $asal_univ;
		$this->email = $email;
	}

// Setter Dan Getter

    // Untuk Asal Universitas
	public function setAsal_univ($asal_univ){
		$this->asal_univ = $asal_univ;
	}

	public function getAsal_univ(){
		return $this->asal_univ;
	}

    // Untuk Email
    public function setEmail($email){
	    $this->email = $email;
	}

	public function getEmail(){
		return $this->email;
	}
}

?>